test/unittests/capi_test.cpp

```diff
@@ -593,7 +593,7 @@ TEST(capi, memory_access)
     memory[0] = 0xaa;
     memory[1] = 0xbb;
 
-    EXPECT_EQ(fizzy_execute(instance, 0, nullptr, 0).value.i64, 0x22bbaa);
+    EXPECT_EQ(fizzy_execute(instance, 0, nullptr, 0).value.i32, 0x22bbaa);
```

Comment:
```suggestion
    EXPECT_EQ(fizzy_execute(instance, 0, nullptr, 0), CResult(0x22bbaa_u32));
```

Reply:
Well the `CResult()` and `Result()` are pretty useless now as they try to compare all 64 bits without knowing the type.